parsely/parsely.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * @package vip-bundle-decoupled
+ */
+
+namespace WPCOMVIP\Decoupled\Parsely;
+
+/**
+ * Registers Parse.ly fields in WPGraphQL.
+ *
+ * @return void
+ */
+function register_parsely_fields() {
+	if ( ! isset( $GLOBALS['parsely'] ) ) {
+		return;
+	}
+
+	register_graphql_field(
```

Comment:
Something we should keep in mind is given this is declared conditionally, we can't add the `schemaOrgMeta` to our `ContentNodeFields` by default, as this will break the code generation part of the build step (as it's based on the schema).

So we either need to always declare this and default to `null` or ask customers to manually add the field to their GraphQL fragment request.